docs/docs/reference/csrf.md

```diff
@@ -38,5 +55,15 @@ from ninja import NinjaAPI
 from ninja.security import django_auth
 
 api = NinjaAPI(auth=django_auth)
+```
+
+
+You can use the Django [ensure_csrf_cookie](https://docs.djangoproject.com/en/4.2/ref/csrf/#django.views.decorators.csrf.ensure_csrf_cookie) decorator on an unprotected route to make it include a `Set-Cookie` header for the CSRF token (note that the route decorator must come upper than the [ensure_csrf_cookie](https://docs.djangoproject.com/en/4.2/ref/csrf/#django.views.decorators.csrf.ensure_csrf_cookie) decorator).
```

Comment:
```suggestion
You can use the Django [ensure_csrf_cookie](https://docs.djangoproject.com/en/4.2/ref/csrf/#django.views.decorators.csrf.ensure_csrf_cookie) decorator on an unprotected route to make it include a `Set-Cookie` header for the CSRF token (note that the route decorator must be executed before (i.e. above) the [ensure_csrf_cookie](https://docs.djangoproject.com/en/4.2/ref/csrf/#django.views.decorators.csrf.ensure_csrf_cookie) decorator).
```

Reply:
Are you sure what is described here works? In my experience, no decorator doing work in `process_response` works(ref #544).

This might very well have changed. But is there perhaps e.g. a test that could be pointed to?

Reply:
Yes, it does work, but only with `HttpResponse`, not a dict. It is true it is unclear, I will update my MR later.